cookbooks/aws-parallelcluster-platform/attributes/platform.rb

```diff
@@ -11,7 +11,9 @@
 
 # Enroot + Pyxis
 default['cluster']['enroot']['version'] = '3.4.1'
+default['cluster']['enroot']['enabled'] = 'no'
```

Comment:
Why `no` rather than `false`?
I know all the attributes in our cookbook are processed as strings, not boolean, but it would be better to use 'true/false' rather than 'true/no'